src/ugene_globals.pri

```diff
@@ -55,6 +57,19 @@ macx {
 }
 
 linux-g++ {
+    # Try to build glibc_2.17 compatible binaries (Ubuntu 14.04/Debian 8/CentOS 7) regardless of the local 'glibc'
+    # version on the build machine. See https://github.com/wheybags/glibc_version_header
+    #
+    # For a wider range of supported platforms this value should be aligned with the 'glibc'
+    # used to build QT binaries.
+    #
+    # This solution is not bulletproof, because it won't replace methods from a newer 'glibc'
+    # that have no older counterparts. To address 'new API' problem we should either use a post-build 'glibc'
+    # version check for all binaries we build or/and run our pre-release tests on the old Ubuntu 16.04 host with
+    # the binary we want to release.
+    QMAKE_CFLAGS += -include $$ROOT_SRC_DIR/include/3rdparty/force_link_glibc_2.17.h
```

Comment:
I think, it's better to locate this file as "ugene/src/libs_3rdparty/glibc/force_link_glibc_2.17.h"

Reply:
Good idea. Done